bin/resin-preload

```diff
@@ -8,9 +8,12 @@ const info = require('../package.json')
 const Promise = require('bluebird')
 const streamToPromise = require('stream-to-promise')
 const resinSdk = require('resin-sdk')
+const tmp = Promise.promisifyAll(require('tmp'))
 
 const docker = new Docker({Promise})
 
+tmp.setGracefulCleanup()
```

Comment:
I would not recommend using `tmp`, as its cleanup doesn't actually work in most cases (as we found out in drivelist), see https://github.com/resin-io-modules/drivelist/pull/191 and https://github.com/resin-io/etcher/issues/1571